File: 01-programming/csharp/snippets/operators_math.cs
using System;
class operators_math{
  static void Main(){
    Console.WriteLine("Please enter a number");
    string input = Console.ReadLine();
    Console.WriteLine("Please enter another number");
    string input2 = Console.ReadLine();
   
    int num = Convert.ToInt32(input);
    int num2 = Convert.ToInt32(input2);

    Console.WriteLine(num+ " + "+num2+" = "+(num+num2));
    Console.WriteLine(num+ " - "+num2+" = "+(num-num2));
    Console.WriteLine(num+ " * "+num2+" = "+(num*num2));
    Console.WriteLine(num+ " / "+num2+" = "+(num/num2));
  }
}

File: 01-programming/haskell/snippets/operators_math.hs
main :: IO ()
main = do
  putStrLn "Enter a number: "
  input <- getLine
  putStrLn "Enter another number: "
  input2 <- getLine
  
  let num = read input::Double
      num2 = read input2::Double

  putStrLn (input++" + "++input2++" = "++show(num+num2))
  putStrLn (input++" - "++input2++" = "++show(num-num2))
  putStrLn (input++" * "++input2++" = "++show(num*num2))
  putStrLn (input++" / "++input2++" = "++show(num/num2))
